### PR TITLE
*: add Tolerations on pod

### PR DESCRIPTION
**Pull request by xiang90**
@hongchaodeng @hasbro17 PTAL

/cc @aaronlevy

### CODE REVIEW

**Comment by xiang90** [comment]
fix https://github.com/coreos/etcd-operator/issues/968

**Comment by aaronlevy** [comment]
LGTM!

**Comment by xiang90** [comment]
@hongchaodeng i am going to test it manually today.

**Comment by xiang90** [comment]
working ok.

File: pkg/spec/cluster.go [hunk 1]
```diff
@@ -127,6 +127,9 @@ type PodPolicy struct {
 	// Resources is the resource requirements for the etcd container.
 	// This field cannot be updated once the cluster is created.
 	Resources v1.ResourceRequirements `json:"resources"`
+
+	// Tolerations specifies the pod's tolerations.
+	Tolerations []v1.Toleration
```
Comment (hongchaodeng) [inline pkg/spec/cluster.go]: json tags?

Comment (xiang90) [inline pkg/spec/cluster.go]: will fix.

**Pull request merged**